test/test_core.py

```diff
@@ -6850,7 +6850,7 @@ def test_poppler(self):
     };
     Module.postRun = () => {
       var FileData = Array.from(MEMFS.getFileDataAsTypedArray(FS.root.contents['filename-1.ppm']));
-      out("Data: " + JSON.stringify(FileData.map(function(x) { return unSign(x, 8) })));
+      out("Data: " + JSON.stringify(FileData));
```

Comment:
Can you add `Data: ` to the second arg to `do_run` below just to confirm this is the string being matched.